8.x/getting-started/contributing.md

```diff
@@ -385,6 +385,115 @@ ___
 
 **10.** For every document that is written and placed in the documentation section of the Larammerce site, the members of the Larammerce team are obliged to read it.
 
+## PR review
+
+This section describes how to run, test and review your teammate's pull request in your local system.
```

Comment:
This section describes how to run, test, and review your teammate's pull request with an editor on your local system.

- Add a comma after `test`.
- Add `with an editor` to the sentence.
- change `in` your local system to `on` your local system.